src/SDKs/CognitiveServices/dataPlane/Knowledge/QnAMaker/QnAMaker/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services QnAMaker APIs.</Description>
+    <Version>2.2.0-preview</Version>
```

Comment:
Why starts with 2.2.0? not something like 0.1.0 or 1.0.0?

Reply:
used the same as face sdk. Fixed.